it/controllers/employment/EmployerLeaveDateControllerISpec.scala

```diff
@@ -0,0 +1,927 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.employment
+
+import forms.employment.EmploymentDateForm
+import models.User
+import models.mongo.{EmploymentCYAModel, EmploymentDetails, EmploymentUserData}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.HeaderNames
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.WSResponse
+import utils.{EmploymentDatabaseHelper, IntegrationTest, ViewHelpers}
+
+import java.time.LocalDate
+
+//scalastyle:off
```

Comment:
is scalastyle:off still needed?

Reply:
kept it to stop the highlighting, scalastyle being turned off is much more of concern in the actual code imo